src/core/lv_obj_style.c

```diff
@@ -924,4 +932,12 @@ static void fade_in_anim_ready(lv_anim_t * a)
     lv_obj_remove_local_style_prop(a->var, LV_STYLE_OPA, 0);
 }
 
-
+static bool is_transform_style(lv_style_prop_t prop)
```

Comment:
Could you handle it by adding a new `LV_STYLE_PROP_FLAG_TRANSFORM` similarly to [these](https://github.com/lvgl/lvgl/blob/master/src/misc/lv_style.c#L41).

Reply:
great, let me do it

Reply:
done